cmd/hope/utils/resources.go

```diff
@@ -66,8 +70,41 @@ func GetIdentifiableResources(names *[]string, tags *[]string) (*[]hope.Resource
 
 	// If any name wasn't found, error out.
 	if len(nameMap) != 0 {
-		return nil, fmt.Errorf("Failed to find resources with names: %s", strings.Join(*maputil.MapStringBoolKeys(&nameMap), ","))
+		return nil, fmt.Errorf("failed to find resources with names: %s", strings.Join(*maputil.MapStringBoolKeys(&nameMap), ","))
 	}
 
 	return &returnSlice, nil
 }
+
+// For each parameter from a file, load the file and populate the base64
+//   values of the files into the properties.
+// Does nothing to deduplicate keys.
+// All plain parameters will exist in the list before file parameters.
+func RenderParameters(directParameters, fileParameters []string) ([]string, error) {
+	rv := directParameters
+
+	for _, param := range fileParameters {
+		paramComponents := strings.SplitAfterN(param, "=", 2)
+		paramName := strings.TrimRight(paramComponents[0], "=")
+		paramPath := paramComponents[1]
+
+		if stat, err := os.Stat(paramPath); err != nil {
+			return nil, err
+		} else if stat.IsDir() {
+			return nil, fmt.Errorf("cannot resolve parameter contents from directory: %s", paramPath)
+		}
+
+		srcFile, err := ioutil.ReadFile(paramPath)
```

Comment:
This probably needs to be replaced with using the parameter substitution system itself. Wouldn't be out of the ordinary for these files to rely on the other parameters.